### PR TITLE
Independant name generator

### PR DESCRIPTION
**Pull request by X-Ryl669**
This is based on #64, but completely reworked on top of #91.

There is now a name generator instance per BaseLibrary instance, so there is plenty more selectors to create.
This requires an major version bump, since the API evolved.

### CODE REVIEW

**Comment by JPeer264** [comment]
Awesome, I'm gonna review it as soon as the conflicts are resolved, then it's a bit easier :) 
Conflicts are easy tho 👍

**Comment by X-Ryl669** [comment]
There's no conflict anymore!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rcs-core",
-  "version": "2.6.2",
```
Comment (JPeer264) [inline package.json]: Please do not increment the version here, as I have an own workflow for that :)

**Comment by X-Ryl669** [comment]
Please increment the version after this one, as I've made some changes in `rename-css-selectors` and I need the version to be different for them to work (and I'll be able to submit the pull request to adapt to the API change). Thank you.

**Pull request merged**